sample-lnd.conf

```diff
@@ -1023,6 +1023,9 @@ litecoin.node=ltcd
 ; (default: 1000)
 ; routerrpc.maxmchistory=900
 
+; The time interval with which the MC store state is flushed to the DB.
```

Comment:
nit: We usually add an example (or the default value, there's no consensus :) ) so users see what value type is expected. And for duration types we also have `Valid time units are {s, m, h}.` in most places.

Reply:
Sorry forgot to finish typing the line for some reason :sweat_smile: done